src/libgv/actor.c

```diff
@@ -201,13 +260,20 @@ void GV_InitActor(int level, GV_ACT *actor, TActorFreeFunction free_func)
     actor->free = free_func;
 }
 
-GV_ACT *GV_NewActor(int level, int memSize)
+/**
+ * @brief Allocate memory for an actor and initialize it.
+ *
+ * @param actorListId The id of the list where the actor will be added.
+ * @param memSize The size of the actor.
+ * @return GV_ACT* The allocated actor.
+ */
+GV_ACT *GV_NewActor(int actorListId, int memSize)
```

Comment:
Actors pass a macro called EXEC_LEVEL to GV_NewActor. Calling this actorListId implies something different imo. The actual list is supposed to be a private implementation detail in actor.c anyway.

Reply:
I will rename to execLevel